src/main/java/pageobjectsnew/IndexPageNew.java

```diff
@@ -0,0 +1,274 @@
+
+    public IndexPageNew openSite() {
+        //get index page
+        getURL(Constants.URL, driver);
+        return this;
+    }
+
+    public IndexPageNew clickOnCreateAccountLink() {
+        waitForElementVisible(createAccountLinkCss, driver);
+        clickOnElement(createAccountLinkCss, "Create account link", driver);
+        return this;
+    }
+
+    public String getPageTitleText(){
+        return pageTitle.getText();
+    }
+
+    public String getErrorText(){
+        return loginErrorMessage.get(0).getText();
+    }
+
+    public IndexPageNew login(User user, String method){
+        sendKeysClear(email, "Email form", user.getEmail(), driver);
+        sendKeysClear(password, "Password form", user.getPassword(), driver);
+
+        switch (method){
+            case "button":
+                clickOnSignInButton();
+                break;
+            case "enter":
+                password.sendKeys(Keys.ENTER);
+                break;
+            default:
+                clickOnSignInButton();
+                break;
+        }
+        return this;
+    }
+
+    public IndexPageNew createAccount(User user){
+        waitForElementVisible(createAccountButtonCss, driver);
+
+        sendKeysClear(emailCreate, "Email form create user", user.getEmail(), driver);
+
+        sendKeysClear(passwordCreate, "Password form create user", user.getPassword(), driver);
+
+        sendKeysClear(passwordConfirmCreate, "Second password form create user", user.getPassword(), driver);
+
+        clickOnCreateAccountButton();
+
+        return this;
+    }
+
+    public IndexPageNew clickOnSignInButton() {
+        waitForElementVisible(signInButtonCss, driver);
+        clickOnElement(signInButtonCss, "SignIn button", driver);
+        return this;
+    }
+
+    public IndexPageNew clickOnCreateAccountButton() {
+        waitForElementVisible(createAccountButtonCss, driver);
+        clickOnElement(createAccountButtonCss, "Create account button", driver);
+        return this;
+    }
+
+    public IndexPageNew addRandomItemsFromIndexPageToCart(int item) {
+
+        ItemPage itemPage = initElements(driver, ItemPage.class);
+
+            for (int i = 0; i < item; i++) {
+                Random random = new Random();
+                int randomItem = random.nextInt(featuredItems.size());
+
+                waitForElementVisible(featuredItems.get(randomItem), driver);
+                clickOnElement(featuredItems.get(randomItem), "Item page", driver);
+
+                try {
+                    itemPage.clickOnAddToCartButton();
+                    waitForElementVisible(itemPage.addToCartPopUp, driver);
+                    getURL(Constants.URL, driver);
+                }
+                catch (TimeoutException e) {
+                      getURL(Constants.URL, driver);
+                      i--;
+                }
+            }
+        return this;
+        }
+
+    public IndexPageNew addRandomItemFromIndexPageToCart(Item item) {
```

Comment:
First method adds a few random items to cart. And this method adds only one random item, but adds its name to item business object. I'll refactor name of second method